tests/ui/panic-runtime/unwind-tables-target-required.rs

```diff
@@ -1,7 +1,7 @@
 // Tests that the compiler errors if the user tries to turn off unwind tables
 // when they are required.
 //
-// only-x86_64-windows-msvc
+// only-x86_64-pc-windows-msvc
```

Comment:
note that I already did this one in https://github.com/rust-lang/rust/pull/108809 and it was not enough